src/sql/src/parsers/copy_parser.rs

```diff
@@ -39,10 +41,74 @@ impl<'a> ParserContext<'a> {
                     actual: self.peek_token_as_string(),
                 })?;
 
-        self.parser
-            .expect_keyword(Keyword::TO)
+        let direction = self
+            .parser
+            .expect_one_of_keywords(&[Keyword::TO, Keyword::FROM])
+            .context(error::SyntaxSnafu { sql: self.sql })?;
+
+        match direction {
+            Keyword::TO => self.parse_copy_table_to(table_name),
+            Keyword::FROM => self.parse_copy_table_from(table_name),
+            _ => unreachable!(),
+        }
```

Comment:
Could we use [parse_keyword](https://docs.rs/sqlparser/latest/sqlparser/parser/struct.Parser.html#method.parse_keyword) and `if-else` here? Using `expect_one_of_keywords` and then matching with an `unreachable` arm is a little verbose.